src/lib/y2partitioner/actions/controllers/md.rb

```diff
@@ -123,10 +131,20 @@ def md_name=(name)
         end
 
         # Title to display in the dialogs during the process
+        #
+        # @param action [Symbol] action currently being performed (:add, :resize)
         # @return [String]
-        def wizard_title
-          # TRANSLATORS: dialog title. %s is a device name like /dev/md0
-          _("Add RAID %s") % md.name
+        def wizard_title(action: nil)
```

Comment:
Couldn't the action been set in the constructor? If an md is passed to `initialize`, then it's `:resize`. So we can avoid this argument here and in the dialogs. Just an idea.